server/env.ts

```diff
@@ -495,6 +495,39 @@ export class Environment {
     process.env.SOURCE_COMMIT || process.env.SOURCE_VERSION
   );
 
+  /**
+   * A boolean switch to toggle the rate limiter
+   * at application web server.
+   */
+  @IsOptional()
+  @IsBoolean()
+  public RATE_LIMITER_ENABLED = this.toBoolean(
+    process.env.RATE_LIMITER_ENABLED ?? "false"
+  );
+
+  /**
+   * Set max allowed requests in a given duration for
+   * default rate limiter to trigger throttling.
+   */
+  @IsOptional()
+  @IsNumber()
+  @CannotUseWithout("RATE_LIMITER_ENABLED")
+  public RATE_LIMITER_REQUESTS = this.toOptionalNumber(
+    process.env.RATE_LIMITER_REQUESTS ?? "5000"
+  );
```

Comment:
```suggestion
  public RATE_LIMITER_REQUESTS = this.toOptionalNumber(
    process.env.RATE_LIMITER_REQUESTS
  ) ?? 5000;
```

Maybe change these two numbers so they are not typed as optional, like: https://github.com/outline/outline/blob/9d792ef1cfb5082860de7eee78d14e820c131278/server/env.ts#L241

Reply:
Fixed!